pkg/inject/inject.go

```diff
@@ -774,73 +783,73 @@ func (conf *ResourceConfig) applyAnnotationOverrides(values *l5dcharts.Values) {
 		}
 	}
 
-	if override, ok := conf.getOverride(k8s.ProxyCPURequestAnnotation); ok {
-		res, err := k8sResource.ParseQuantity(override)
+	if override, ok := annotations[k8s.ProxyCPURequestAnnotation]; ok {
+		_, err := k8sResource.ParseQuantity(override)
 		if err != nil {
 			log.Warnf("%s (%s)", err, k8s.ProxyCPURequestAnnotation)
 		} else {
-			values.Global.Proxy.Resources.CPU.Request = res.String()
+			values.Global.Proxy.Resources.CPU.Request = override
 		}
 	}
 
-	if override, ok := conf.getOverride(k8s.ProxyMemoryRequestAnnotation); ok {
-		res, err := k8sResource.ParseQuantity(override)
+	if override, ok := annotations[k8s.ProxyMemoryRequestAnnotation]; ok {
+		_, err := k8sResource.ParseQuantity(override)
 		if err != nil {
 			log.Warnf("%s (%s)", err, k8s.ProxyMemoryRequestAnnotation)
 		} else {
-			values.Global.Proxy.Resources.Memory.Request = res.String()
+			values.Global.Proxy.Resources.Memory.Request = override
 		}
 	}
 
-	if override, ok := conf.getOverride(k8s.ProxyCPULimitAnnotation); ok {
-		res, err := k8sResource.ParseQuantity(override)
+	if override, ok := annotations[k8s.ProxyCPULimitAnnotation]; ok {
+		_, err := k8sResource.ParseQuantity(override)
```

Comment:
I think we should still have this logic right i.e to convert into k8s resource quanitites and pass them into rendering?

Also, This is the reason why `inject_test.go` has to be updated with resources in different formatting.

Reply:
I could potentially go either way on this, but it seems to me if someone puts a certain string in the annotation then that's the string we should use in the template.  It's good for us to do validation so that we can warn if it's invalid, but there's not really any reason why we need to do normalization or change the format of it.

Basically, it's easier to understand what's happening here if we just use the string we're given and avoid changing it for no reason.

Reply:
Yep, That makes sense! But would be not break existing using behavior i.e values that were set without conversion by doing this?